src/bundle/Resources/public/js/scripts/sidebar/btn/content.edit.js

```diff
@@ -0,0 +1,21 @@
+(function () {
+    const editButton = document.querySelector('.ez-btn--edit[data-actions="edit"]');
+
+    editButton.addEventListener('click', () => {
+        const languageButtons = document.querySelectorAll('.ez-extra-actions--edit[data-actions="edit"] .form-check [type="radio"]');
```

Comment:
The selector looks very specific, can't it be simpler? The second thing is that `languageButtons` should be defined in the scope above. So it will be cached once and it will be always available. Otherwise, you're looking for it everytime you're clicking on a button.